DecipherNow/gm-fabric-dashboard
# Fix: Added sort by latency to JVM routes table

Resolves #1272 
  
## How to Test

  - Navigate to a JVM routes table and confirm sort by latency 50% and 99% works as expected.
